client/js/analysis.js

```diff
@@ -1,6 +1,16 @@
 const COMMENT_APPEARANCE_TIME = 1500;
 const COMMENT_TO_STOP_AT = 5;
 const FEEDBACK_APPEARANCE_TIME = 1500;
+const COOLDOWN_TIME = 1000;
+const TEN_MINUTES_IN_SECONDS = 3600;
```

Comment:
there are 600 seconds in 10 minutes, not 3600. also you should replace the number with the equation that resulted in that number (60 * 10)